### ISSUE TITLE
Fix random seed

### ISSUE THREAD

**Issue by corentinlelannier** [vertica/VerticaPy#92]
Hello,

I try to fix a random seed with "set_option("random_state",2)" to get the same train and test datasets at each exection of the code but the result changes.

The parameter SEEDED_RANDOM given in the SQL function changes each time.

`set_option("random_state",2)`
`train,test = titanic.train_test_split()`
`train_1,test_1 = titanic.train_test_split()`
`print(train.current_relation())`
`print(train_1.current_relation())`

Regards

**Comment by oualib** [comment]
Hi Corentin,

As Vertica is a relational DB, the only way to seed the randomness is to order the data. As specified in the doc, the parameter: order_by will help you to seed the randomness.

order_by: dict / list, optional
        List of the vcolumns to use to sort the data using asc order or
        dictionary of all the sorting methods. For example, to sort by "column1"
        ASC and "column2" DESC, write {"column1": "asc", "column2": "desc"}
        **Without this parameter, the seeded random number used to split the data
        into train and test can not garanty that no collision occurs. Use this
        parameter to avoid collisions.**

Can you please try:

train,test = titanic.train_test_split(order_by = ["name"])
train_1,test_1 = titanic.train_test_split(order_by = ["name"])

**Comment by corentinlelannier** [comment]
Hello,

I tried with the parameter order_by but I get the same result.
I also tried with iris :

`unique = iris.groupby(columns = ["SepalLengthCm","SepalWidthCm","PetalLengthCm","PetalWidthCm"])`
`iris_train_0,iris_test_0 = unique.train_test_split(order_by = ["SepalLengthCm","SepalWidthCm","PetalLengthCm","PetalWidthCm"])`
`iris_train_1,iris_test_1 = unique.train_test_split(order_by = ["SepalLengthCm","SepalWidthCm","PetalLengthCm","PetalWidthCm"])`

`print(iris_train_0.current_relation())`
`print(iris_train_1.current_relation())`
`print(iris_train_0["SepalLengthCm"].sum())`
`print(iris_train_1["SepalLengthCm"].sum())`

The parameters SEEDED_RANDOM in SQL and the sums change.

Regards

**Comment by oualib** [comment]
Thanks Corentin for the information. The function is now corrected and it will be available in the next release. In the current version, you can still use the parameter 'random_state' of the function to seed the randomness.

**Comment by oualib** [comment]
This issue is solved in the version 0.6.0
